src/templates/FundraiserPage.tsx

```diff
@@ -0,0 +1,14 @@
+import { Fundraiser } from '@components/fundraiser/Fundraiser'
```

Comment:
NOTE: Use Gatsby's filesystem route API to generate pages, instead of templates.

See [#623](https://github.com/distributeaid/distributeaid.org/issues/623) for context.